### ISSUE TITLE
Runs on attributes not containing links, triggers cachebust on `undefined`

### ISSUE THREAD

**Issue by elfuego** [furzeface/gulp-cache-bust#7]
Consider the following code example (don't mind the incorrect ng-if ;-) 

``` html
<button class="forward" ng-if="undefined"></button>
```

It yields the following result :

``` html
<button class="forward" ng-if="undefined?t=1448621108414"></button>
```

Replacing `undefined` with `false` (or any other string I tested) does not have this effect.

**Comment by tbredin** [comment]
+1 This is breaking for me in an inline script tag on "undefined"

**Comment by elfuego** [comment]
This can be fixed by including version 2.0.0 of [cachebust](https://github.com/furzeface/cachebust) I found out.

**Comment by davinkevin** [comment]
Seems to be fixed in v1.1.0. Could you confirm and close this issue ?

**Comment by davinkevin** [comment]
No confirmation, close this issue
